homeassistant/components/climate/sensibo.py

```diff
@@ -106,12 +106,13 @@ def async_assume_state(service):
 class SensiboClimate(ClimateDevice):
     """Representation of a Sensibo device."""
 
-    def __init__(self, client, data):
+    def __init__(self, hass, client, data):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.

Reply:
Move the first `_do_update`call to `async_added_to_hass`.

Reply:
`_do_update` needs `data` and I was just trying to avoid storing `data` as an instance variable since a new version gets fetched for each update and passed to `_do_update`. Should I just store `data` during `__init__` and to pass in to the first `_do_update` call inside `async_added_to_hass`?

Reply:
Maybe easier to pass in the temp unit to the instance and store that as an attribute. Then we don't need to move first call of `_do_update`.

Reply:
I like that better. Thanks